### ISSUE TITLE
org.specs.specification.PathException: TreePath(List(0, 0, 1))not found for have no repetition on different pages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#109]
```
I got the following when running a test suite with specs 1.6.1 (I was
previously using 1.5.0 without issues). Any ideas?

org.specs.specification.PathException: TreePath(List(0, 0, 1))not found for
have no repetition on different pages
    at
org.specs.specification.SpecificationExecutor$class.executeExample(Specification
Executor.scala:47)
    at
org.specs.specification.BaseSpecification.executeExample(BaseSpecification.scala
:55)
    at
org.specs.specification.BaseSpecification.executeExample(BaseSpecification.scala
:55)
    at
org.specs.specification.ExampleLifeCycle$$anonfun$executeExample$1.apply(Example
LifeCycle.scala:124)
    at
org.specs.specification.ExampleLifeCycle$$anonfun$executeExample$1.apply(Example
LifeCycle.scala:124)
    at scala.Option.map(Option.scala:70)
    at
org.specs.specification.ExampleLifeCycle$class.executeExample(ExampleLifeCycle.s
cala:124)
    at org.specs.specification.Examples.executeExample(Examples.scala:52)
    at org.specs.specification.Examples.executeExample(Examples.scala:52)
    at
org.specs.specification.Examples$$anonfun$executeExamples$1.apply(Examples.scala
:80)
    at
org.specs.specification.Examples$$anonfun$executeExamples$1.apply(Examples.scala
:80)
    at scala.Option.map(Option.scala:70)
    at org.specs.specification.Examples.executeExamples(Examples.scala:80)
    at
org.specs.specification.ExampleStructure$class.examples(ExampleStructure.scala:7
7)
    at org.specs.specification.Examples.examples(Examples.scala:52)
    at
org.specs.runner.ExamplesTestSuite$$anonfun$initialize$4.apply(JUnit.scala:157)
    at
org.specs.runner.ExamplesTestSuite$$anonfun$initialize$4.apply(JUnit.scala:153)
    at scala.List.foreach(List.scala:841)
    at org.specs.runner.ExamplesTestSuite.initialize(JUnit.scala:153)
    at org.specs.runner.JUnitSuite$class.init(JUnit.scala:53)
    at org.specs.runner.ExamplesTestSuite.init(JUnit.scala:143)
    at org.specs.runner.JUnitSuite$class.getName(JUnit.scala:65)
    at org.specs.runner.ExamplesTestSuite.getName(JUnit.scala:143)
    at
org.specs.runner.TestDescription$class.asDescription(JUnitSuiteRunner.scala:116)
    at org.specs.runner.JUnitSuiteRunner.asDescription(JUnitSuiteRunner.scala:32)
    at
org.specs.runner.TestDescription$class.makeDescription(JUnitSuiteRunner.scala:12
5)
    at
org.specs.runner.JUnitSuiteRunner.makeDescription(JUnitSuiteRunner.scala:32)
    at
org.specs.runner.TestDescription$$anonfun$makeDescription$1.apply(JUnitSuiteRunn
er.scala:127)
    at
org.specs.runner.TestDescription$$anonfun$makeDescription$1.apply(JUnitSuiteRunn
er.scala:126)
    at scala.List.foreach(List.scala:841)
    at
org.specs.runner.TestDescription$class.makeDescription(JUnitSuiteRunner.scala:12
6)
    at
org.specs.runner.JUnitSuiteRunner.makeDescription(JUnitSuiteRunner.scala:32)
    at org.specs.runner.JUnitSuiteRunner.getDescription(JUnitSuiteRunner.scala:57)
    at
org.junit.internal.runners.CompositeRunner.getDescription(CompositeRunner.java:4
0)
    at
org.junit.internal.runners.CompositeRunner.getDescription(CompositeRunner.java:4
0)
    at org.junit.runner.Runner.testCount(Runner.java:38)
    at
org.eclipse.jdt.internal.junit4.runner.JUnit4TestClassReference.countTestCases(J
Unit4TestClassReference.java:30)
    at
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.countTests(RemoteTestRunn
er.java:487)
    at
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner
.java:455)
    at
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner
.java:683)
    at
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java
:390)
    at
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.jav
a:197)

```

Original issue reported on code.google.com by `ismaelj` on 17 Jan 2010 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
I've recently seen this issue in 
http://code.google.com/p/specs/issues/detail?id=103.

Is it the same case? If not, can you send me a short spec reproducing the issue?

Thanks,

Eric.
```

Original comment by `etorrebo...@gmail.com` on 17 Jan 2010 at 9:19

**Comment by GoogleCodeExporter** [comment]
```
Weird, I thought I had searched for PathException. Maybe I had the filter 
incorrectly
configured to only show open issues. I'll try with specs 1.6.2-SNAPSHOT and see 
if
that helps and will report back.
```

Original comment by `ismaelj` on 17 Jan 2010 at 9:25

**Comment by GoogleCodeExporter** [comment]
```
Seems like this is fixed in 1.6.2-SNAPSHOT indeed. Please close as a duplicate 
of 
issue #103. Thanks.
```

Original comment by `ismaelj` on 19 Jan 2010 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
I forgot to ask, any idea when 1.6.2 will be released? It would also be nice if 
that 
included a resolution for Issue #110. It's painful to have to disable detailed 
diffs 
manually for each case where it takes too long.
```

Original comment by `ismaelj` on 19 Jan 2010 at 7:00

**Comment by GoogleCodeExporter** [comment]
```
I'll try to fix issue 110 and to release 1.6.2 this week.
```

Original comment by `etorrebo...@gmail.com` on 26 Jan 2010 at 3:21
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 31 Mar 2011 at 8:05
- Added labels: **mutable**
